evergreen.yml

```diff
@@ -251,6 +251,15 @@ functions:
         bucket: mciuploads
         permissions: public-read
         content_type: application/octet-stream
+    - command: s3.put
+      params:
```

Comment:
The filename for this on evergreen is `connector.mez` I suppose it uses the original filename.  Let's update that to `MongoDBAtlasODBC.mez` by adding a parameter:
```
display_name: "MongoDBAtlasODBC.mez"
```

Reply:
Great idea!